rust/lance-linalg/benches/dot.rs

```diff
@@ -92,6 +96,27 @@ fn bench_distance(c: &mut Criterion) {
         });
     });
 
+    let mut rng = rand::thread_rng();
+    let key = repeat_with(|| rng.gen::<f32>())
+        .map(bf16::from_f32)
```

Comment:
It probably doesn't matter too much but I think this will result in a lot of `0` values.

Reply:
i See. what is good way to prevent that?